### PR TITLE
Fix the bug related to APA-31469. Quote field names with reserved key…

### PR DESCRIPTION
**Pull request by kxu1026**
…words in PrestoSQL

### CODE REVIEW

File: coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/PrestoTryCastFunction.java [hunk 1]
```diff
@@ -42,5 +42,4 @@ public void unparse(SqlWriter writer, SqlCall call, int leftPrec, int rightPrec)
     call.operand(1).unparse(writer, 0, 0);
     writer.endFunCall(frame);
   }
-
```
Comment (cwsteinbach) [inline coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/PrestoTryCastFunction.java:45]: Revert whitespace change.

File: coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/GenericProjectToPrestoConverter.java [hunk 2]
```diff
@@ -354,6 +354,4 @@ private static String buildStructRelDataTypeFieldAccessString(RelRecordType from
     String subFieldsString = String.join(", ", structSelectedFieldStrings);
     return String.format("row(%s)", subFieldsString);
   }
-
```
Comment (cwsteinbach) [inline coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/GenericProjectToPrestoConverter.java:357]: Revert whitespace change.

File: coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/PrestoElementAtFunction.java [hunk 3]
```diff
@@ -25,7 +25,6 @@
 import org.apache.calcite.sql.type.SqlTypeName;
 
 
-
```
Comment (cwsteinbach) [inline coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/PrestoElementAtFunction.java:28]: Revert whitespace change.

File: coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/PrestoKeywordsConverter.java [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+   * @return if the value is a reserved keyword, a double quote will be added as a return value.
+   */
+  public static String quoteReservedKeyword(String value) {
+    if (RESERVED_KEYWORDS.containsKey(value.toUpperCase())) {
+      return "\"" + value + "\"";
+    } else {
+      return value;
+    }
+  }
+}
```
Comment (cwsteinbach) [inline coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/PrestoKeywordsConverter.java:109]: Add newline.

File: coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/PrestoKeywordsConverter.java [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+package com.linkedin.coral.presto.rel2presto.functions;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+
+/**
+ * Presto Keywords Converter maintains the Presto Keywords and related utility functions
+ */
+public final class PrestoKeywordsConverter {
```
Comment (cwsteinbach) [inline coral-presto/src/main/java/com/linkedin/coral/presto/rel2presto/functions/PrestoKeywordsConverter.java:15]: Please consider replacing this class with an enum.

**Comment by kxu1026** [comment]
this is a duplicate PR. dropping the request.

**Pull request closed without merging**